client/components/NavBar.js

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+
+import Notification from './Notification';
+
+export class NavBar extends React.Component {
+  constructor(props){
+    super(props);
+
+    this.logOut = this.logOut.bind(this);
+  }
+  /**
+   * [logOut description]
+   * @method logOut
+   * @return {[type]} [description]
+   */
+  logOut() {
+    window.localStorage.removeItem('token');
+    window.location.href = '/';
+    alert(1234);

```

Comment:
Why do you have an alert after you've logged the user out? Besides, your alert is not descriptive.